remotespark/livyclientlib/sparkcontroller.py

```diff
@@ -72,6 +72,7 @@ def add_session(self, name, connection_string, skip_if_exists, properties):
         session.start()
         livy_client = self.client_factory.build_client(session)
         self.client_manager.add_client(name, livy_client)
+        livy_client.start()
```

Comment:
I don't know how it would work if it went in the factory.  The job of a factory is to create other objects, not engage some behaviors on them.